mempool/mempool.go

```diff
@@ -393,6 +394,8 @@ func (mp *TxPool) RemoveDoubleSpends(tx *btcutil.Tx) {
 			}
 		}
 	}
+
```

Comment:
For consistency with the rest of the code and relevant changes, please wrap the lock unlock around the critical section without blank lines.

```Go
func (mp *TxPool) RemoveDoubleSpends(tx *btcutil.Tx) {
	mp.mtx.Lock()
	for _, txIn := range tx.MsgTx().TxIn {
		...
	}
	mp.mtx.Unlock()
}